.test/config/config.yaml

```diff
@@ -16,6 +16,7 @@ krakenuniq_db: resources/KrakenUniq_DB
 bowtie2_patho_db: resources/ref.fa
 pathogenome_path: resources
 pathogenomesFound: resources/pathogenomesFound.tab
+pathogenome_seqid2taxid_db: resources/seqid2taxid.pathogen.map
```

Comment:
This would require an addition to [config.schema.yaml (e.g. before L54)](https://github.com/NBISweden/ancient-microbiome-smk/blob/main/workflow/schemas/config.schema.yaml#L54). Also, this would be a good moment to eliminate the pathogenome_path key and add:

`params:
    pathogenome_path=os.path.dirname(config["pathogenomesFound"])`

to the mapDamage rule

Reply:
Hmmm, for this toy data set, the files

pathogenome_seqid2taxid_db: resources/seqid2taxid.pathogen.map
malt_seqid2taxid_db: resources/KrakenUniq_DB/seqid2taxid.map

can be identical, yes. However, I think they should not be for real world projects. Basically, we should have different seqid2taxid for malt, krakenuniq and pathogenome. I am not sure whether everything will work as expected if we have identical seqid2taxid for all the three tools. In this case it should the NT seqid2taxid because the three tools use databases that are basically different subsets of NT. I built individual seqid2taxid for the three tools, so can't guarantee that a universal seqid2taxid will work as expected

Reply:
Ok, makes sense. Keep the malt_seqidtaxid_db key but add the following to schemas/config.schema.yaml (e.g. before pathogenomesFound key):

`  pathogenome_seqid2taxid_db:
    type: string
    description: >-
      Add description here
`
Some of the other database keys could do with more elaborate descriptions. You have a better idea of what each key means so at some point it would be good if you could amend the descriptions.